<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgDynaOptions</class>
 <widget class="QWidget" name="WdgDynaOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>210</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>359</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dynamics settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="initWidthLbl">
           <property name="text">
            <string>Initial width:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KisDoubleParseSpinBox" name="initWidthSPBox">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Mass:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KisDoubleParseSpinBox" name="massSPBox">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Drag:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KisDoubleParseSpinBox" name="dragSPBox">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.150000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Width range:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KisDoubleParseSpinBox" name="widthRangeSPBox">
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>153</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ShapeTab">
      <attribute name="title">
       <string>Shape</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Diameter:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KisDoubleSliderSpinBox" name="diameterDSSB" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Angle:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KisDoubleSliderSpinBox" name="angleSlider" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="fixedAngleChBox">
           <property name="text">
            <string>Fixed angle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="circleRBox">
             <property name="text">
              <string>C&amp;ircle</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="twoCBox">
             <property name="text">
              <string>Two</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="linesRBox">
             <property name="text">
              <string>Lines</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisDoubleParseSpinBox" name="lineSpacingSPBox">
             <property name="toolTip">
              <string>Line spacing</string>
             </property>
             <property name="value">
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisIntParseSpinBox" name="lineCountSPBox">
             <property name="toolTip">
              <string>Line count</string>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="polygonRBox">
           <property name="text">
            <string>Poly&amp;gon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wireRBox">
           <property name="text">
            <string>Wi&amp;re</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="LineCBox">
           <property name="text">
            <string>Paint connection</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleParseSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>kis_double_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
